src/main/java/com/github/cross_language_cpp/djinni_intellij_plugin/_DjinniLexer.flex

```diff
@@ -59,16 +48,22 @@ TEXT=[a-zA-Z_0-9]+
   "("                   { return LEFT_PARAM_BRACE; }
   ")"                   { return RIGHT_PARAM_BRACE; }
   "@"                   { return AT; }
-  "list"                { return LIST; }
-  "set"                 { return SET; }
-  "optional"            { return OPTIONAL; }
-  "map"                 { return MAP; }
   "enum"                { return ENUM; }
+  "flags"               { return FLAGS; }
+  "all"                 { return ALL; }
+  "none"                { return NONE; }
   "record"              { return RECORD; }
   "eq"                  { return EQ_KEYWORD; }
   "ord"                 { return ORD; }
   "interface"           { return INTERFACE; }
   "static"              { return STATIC; }
+  "const"               { return CONST; }
+  "extern"              { return EXTERN; }
+  "import"              { return IMPORT; }
+  "list"                { return LIST; }
+  "set"                 { return SET; }
+  "optional"            { return OPTIONAL; }
+  "map"                 { return MAP; }
```

Comment:
This document is auto-generated, don't be confused by the many changes. It seems like the generator wanted to sort things differently today :P